src/lib/y2users/config.rb

```diff
@@ -67,6 +73,13 @@ def groups
       groups_collection.dup.freeze
     end
 
+    # Whether the login is configured
+    #
+    # @return [Boolean]
+    def login?
+      !login.nil?
```

Comment:
To be honest I do not like much that spread of nil everywhere for not configured stuff when it can be done in object itself. See e.g. collections there. Why not have it in LoginConfig as attribute like `configured?` so here you can use `login.configured?`

Reply:
Well, having nil value was the agreed decision to indicate that a value is unknown, see https://github.com/yast/yast-users/pull/287. In general, the writers will perform no action for nil attributes. Here, `#login` is following the same approach, so a `nil` login config means that it is unknown, so it should not persisted to the system. If we want to explicitly disable autologin, then we need to configure the login config properly. The same approach is also followed by `#useradd` here https://github.com/yast/yast-users/pull/284.